internal/cli/atlas/deployments/start.go

```diff
@@ -91,6 +91,9 @@ func (opts *StartOpts) RunLocal(ctx context.Context) error {
 }
 
 func (opts *StartOpts) startContainer(ctx context.Context, deployment options.Deployment) error {
+	opts.StartSpinner()
+	defer opts.StopSpinner()
```

Comment:
I think the second requirement was not too clear, but similar to how `setup` cmd works, the start cmd should wait for mongot to be running.
`update the deployment start command to wait for the containers to start before returning a successful message to the user`

Reply:
oh I see, so we wait for mongot to be available?

Reply:
I think pause is almost synchronous as podman sends signals to close the process